LZW/LZW/Program.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Data;
+
+namespace LZW
+{
+    class Program
+    {
+        static void Main()
+        {
+            // Качественно реализовать структуру данных бор
+            // Реализовать класс архиватора и разоархиватора
+            string? pathAndKey;
+          
+            while (true)
+            {
+
+
+                Console.WriteLine("<<<<<This program compresses and decompresses files by LZW algorithm>>>>>\n");
+                Console.Write("Please, enter the absolute path to the file and key:\n-c - to compress the file\n" +
+                    "-u - to decompress the file\nFormat - Path key: ");
+                pathAndKey = Console.ReadLine();
+
+                if (pathAndKey != null && pathAndKey.Split(' ').Length == 2 && (pathAndKey.Split(' ')[1] == "-c" || pathAndKey.Split(' ')[1] == "-u"))
+                {
+                    if (File.Exists(@pathAndKey.Split(' ')[0]))
+                    {
+                        break;
+                    }
+                    Console.WriteLine("Bad path");
+                }
+                else
+                {
+                    Console.WriteLine("Bad key");
+                }
+            }
+
+            string path = pathAndKey.Split(' ')[0];
+            string key = pathAndKey.Split(' ')[1];
+            string? fileText;
+
+            using (StreamReader stream = new StreamReader(path, true))
+            {
+                fileText = stream.ReadToEnd();
+            }
+
+            FileInfo fi = new FileInfo(@path);
```

Comment:
```suggestion
            FileInfo fi = new FileInfo(path);
```
`path` не ключевое слово, так что `@` не нужна